packages/eas-cli/src/analytics.ts

```diff
@@ -20,40 +18,35 @@ const PLATFORM_TO_ANALYTICS_PLATFORM: { [platform: string]: string } = {
   linux: 'Linux',
 };
 
-let amplitudeClient: Amplitude.NodeClient | null = null;
 let rudderstackClient: RudderAnalytics | null = null;
 let userIdentified = false;
 let identifyData: {
   userId: string;
   deviceId: string;
-  identify: Identify;
+  traits: Record<string, any>;
 } | null = null;
 
 export async function initAsync(): Promise<void> {
```

Comment:
This block is the only thing we need to support old settings format:
```suggestion
export async function initAsync(): Promise<void> {
  // TODO: remove after some time
  const amplitudeEnabled = await UserSettings.getAsync('amplitudeEnabled', null);
  if (amplitudeEnabled !== null) {
    await UserSettings.setAsync('analyticsEnabled', amplitudeEnabled);
    await UserSettings.deleteKeyAsync('amplitudeEnabled');
  }
  const amplitudeDeviceId = await UserSettings.getAsync('amplitudeDeviceId', null);
  if (amplitudeDeviceId !== null) {
    await UserSettings.setAsync('analyticsDeviceId', amplitudeDeviceId);
    await UserSettings.deleteKeyAsync('amplitudeDeviceId');
  } 
  // TODO: cut here
```

Then, in the rest of code, you can rely only on the `analyticsEnabled` and `analyticsDeviceId` settings.